From ea5e68de7f39a7637ff94fd192503152ccb82b6e Mon Sep 17 00:00:00 2001
Message-Id: <ea5e68de7f39a7637ff94fd192503152ccb82b6e.1417465742.git.chang-joon.lee@intel.com>
In-Reply-To: <664d811e547fec51f0453c5acda92012de288ecd.1417465742.git.chang-joon.lee@intel.com>
References: <664d811e547fec51f0453c5acda92012de288ecd.1417465742.git.chang-joon.lee@intel.com>
From: Vidya Srinivas <vidya.srinivas@intel.com>
Date: Tue, 18 Nov 2014 20:45:58 +0530
Subject: [PATCH 10/11] FOR_UPSTREAM [VPG]: drm/i915: Use MMIO flip always.

Always use MMIO flip. Using command mode flip causes
flip failures in some scenarios.

Issue: GMINL-4022
Change-Id: Idcf724485626fc96dd71ac0cafc4681b2ddc368a
Signed-off-by: Vidya Srinivas <vidya.srinivas@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 5d4ec13..0670b87 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -52,7 +52,7 @@ struct i915_params i915 __read_mostly = {
 	.enable_cmd_parser = 1,
 	.disable_vtd_wa = 0,
 	.drrs_interval = 2000,
-	.use_mmio_flip = 0,
+	.use_mmio_flip = 1,
 	.hangcheck_period = 1000,
 	.ring_reset_min_alive_period = 0,
 	.gpu_reset_min_alive_period = 0,
@@ -192,7 +192,7 @@ MODULE_PARM_DESC(drrs_interval,
 
 module_param_named(use_mmio_flip, i915.use_mmio_flip, int, 0600);
 MODULE_PARM_DESC(use_mmio_flip,
-		 "use MMIO flips (-1=never, 0=driver discretion [default], 1=always)");
+		 "use MMIO flips (-1=never, 0=driver discretion, 1=always [default])");
 
 int hangcheck_period_set(const char *val, const struct kernel_param *kp)
 {
-- 
1.7.9.5

